lib/datastore.js

```diff
@@ -490,9 +497,12 @@ Datastore.prototype._update = function (query, updateQuery, options, cb) {
 	  return callback(err);
 	}
 
-	// Update the datafile
-    self.persistence.persistNewState(_.pluck(modifications, 'newDoc'), function (err) {
+    // Update the datafile
+	  var newDocs = _.pluck(modifications, 'newDoc');
+    self.persistence.persistNewState(newDocs, function (err) {
       if (err) { return callback(err); }
+      if (newDocs.length > 0)
+        self.emit('updated', newDocs);
```

Comment:
Though not critical (IMHO, anyway), it would be lovely if the `'updated'` event would actually include 2 arguments: the `newDocs` array **AND** an `oldDocs` array.